JASP-Desktop/components/JASP/Controls/JASPGridControl.qml

```diff
@@ -0,0 +1,136 @@
+//
+// Copyright (C) 2013-2018 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+
+
+import QtQuick		2.11
+import JASP.Widgets 1.0
+import JASP			1.0
+
+JASPControl
+{
+	id						: jaspGridControl
+	controlType				: JASPControlBase.VariablesListView
+	background				: itemRectangle
+	implicitWidth 			: parent.width
+	implicitHeight			: itemTitle.height + itemGrid.height + 2 * jaspTheme.contentMargin
+	useControlMouseArea		: false
+	shouldStealHover		: false
+	innerControl			: itemGrid
+
+	onPreferredHeightChanged:
+	{
+		if (preferredHeight != implicitHeight)
+		{
+			// preferredHeight has been specifically set
+			implicitHeight = implicitHeight; // Break binding
+			height = preferredHeight;
+			itemRectangle.height = Qt.binding(function() { return preferredHeight - itemTitle.height; })
```

Comment:
And why do you change this `itemRectangle.height` ? The original binding `height			: jaspGridControl.height - itemTitle.height` gives the same value right?
Because `jaspGridControl.height = jaspGridControl.preferredHeight`, so why change it?

Reply:
It's removed now